contact.php

```diff
@@ -53,8 +53,8 @@
     global $xoopsConfig, $xoopsDB, $myts, $meta;
     require_once XOOPS_ROOT_PATH . '/modules/adslight/include/gtickets.php';
 
-    if (!$xoopsGTicket->check(true, 'token')) {
-        redirect_header(XOOPS_URL . '/modules/adslight/viewads.php?lid=' . addslashes($id) . '', 3, $xoopsGTicket->getErrors());
+    if (!$GLOBALS['xoopsSecurity']->check(true, $_REQUEST['token'])) {
```

Comment:
Suggest removing the 'token' name and using the default instead. I.e.
```php
if (!$GLOBALS['xoopsSecurity']->check()) {
```
and below at 251:
```php
.$GLOBALS['xoopsSecurity']->getTokenHTML() . '
```
It will make it easier to maintain long term.